workers/advisor/src/main/kotlin/advisor/AdvisorWorker.kt

```diff
@@ -44,6 +45,8 @@ internal class AdvisorWorker(
     private val contextFactory: WorkerContextFactory
 ) {
     fun run(jobId: Long, traceId: String): RunResult = runCatching {
+        logVersion()
```

Comment:
Instead of adding the `logVersion()` call to all workers, I think it would be possible to add it to the `EndpointComponent` class. Then the information would also be printed for core and orchestrator.

Reply:
To which function of `EndpointComponenent` should I add it then? It feels `start()` is only called once at initialization.

Also if the logging is done these, will it be correctly filtered by Grafana's workers filter ?

Reply:
But in this case, you would have to go maybe (way) back in the logs to find this information. Isn't it more practical to see it in every worker run ?

Reply:
Being a bit egoistic, for us as users of the Kubernetes transport, this does not make a difference. Here the message is printed for each worker run.

Reply:
Ok. I hope I got it correctly. Please review the changes.